### ISSUE TITLE
JAWS Won't Move Off <label> When Using aria-details Shortcut

### ISSUE THREAD

**Issue by jgw213** [FreedomScientific/standards-support#716]
### Summary

> Brief description of the issue. Please list any specific steps.
> If I have a label element that has the aria-details attribute, the shortcut to move to the element referenced (alt+insert+d), does not work. It will just announce "Failed to move to details." Interestingly, if your virtual cursor is on the element referenced by the label's aria-details attribute, the shortcut will jump you back to the label.
> Example:
> 1. Go to https://jsbin.com/pejucemobo/edit?html,css,js,output
> 2. Arrow to the label
> 3. Use Alt+Insert+D and get the error message

### Expected result

> What did you expect to happen?
The virtual cursor should move to the detail text

### Actual result

> What actually did happen?
It announces "Failed to move to details."

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

https://jsbin.com/pejucemobo/edit?html,css,js,output

#### JAWS version and build number
Version: 2023.2303.144

 
#### Operating System and version
Using latest version of Windows 10

#### Browser and version:
Latest version of Chrome

**Comment by stevefaulkner** [comment]
@jgw213 have confirmed this is a JAWS bug
test case https://codepen.io/stevef/pen/mdzGXjr
works as expected in NVDA

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 17974

**Comment by BrettLewisVispero** [comment]
Tested with JAWS 2024.
